### ISSUE TITLE
About Episodic Life at Test Phase

### ISSUE THREAD

**Issue by zmonoid** [Kaixhin/Rainbow#36]
In quite a lot implementations and papers, the scores reported are actually when the game is over instead of loss of life (episodic life is only used during training).

You may consider remove episodic life for testing environment to match the score reported.

**Comment by Kaixhin** [comment]
I already do this. In `test.py`, [a new environment is created and `env.eval()` is called](https://github.com/Kaixhin/Rainbow/blob/master/test.py#L17-L18). `env.eval()` sets `self.training = False`, which is a flag that when set true [activates the code path for terminating on loss of life](https://github.com/Kaixhin/Rainbow/blob/master/env.py#L68-L74).

**Comment by zmonoid** [comment]
@Kaixhin Thanks for your information.
